session6/com.itschool.course/src/homework/Ex7.java

```diff
@@ -0,0 +1,19 @@
+package homework;
+
+public class Ex7 {
+
+    public static void main(String[] args) {
+
+        int sideOne = 4;
+        int sideTwo = 5;
+        int sideThree = 6;
+
+        if (sideOne == sideTwo && sideTwo == sideThree && sideThree == sideOne){
```

Comment:
`sideThree == sideOne` is not needed, since if `sideOne == sideTwo && sideTwo == sideThree` automatically means that `sideThree == sideOne`.